src/app/use-cases/__test__/request-signature.spec.ts

```diff
@@ -0,0 +1,90 @@
+import { pipe } from "fp-ts/function";
+import { isRight as isValid } from "fp-ts/Either";
```

Comment:
please don't rename this kind of methods otherwise it makes code harder to be read 
(I'd to look back for what `isValid` stands for :- ) just `import * from E` and use `E.isRight`

Reply:
😬 Sorry, I'll fix it immediately. I believe VS code has done this!

Reply:
Sorry, I'll fix it immediately. I inherited this syntax from [here](https://github.com/pagopa/io-functions-sign/blob/6bb7181fce66f4faeca07cbc3d441968efa2a690/src/signature-request/__test__/document.spec.ts#L1)